.travis.yml

```diff
@@ -0,0 +1,19 @@
+dist: bionic
+addons:
+  apt:
+    packages:
+      - cmake
+      - valgrind
+matrix:
+  fast_finish: true
+  include:
```

Comment:
I don't think we need 1.31.0 here right? `finalfusion-ffi` already guarantees that it builds with 1.31.0. Beyond that, it's just symbols from the C/C++ side.